randovania/games/dread/json_data/Ghavoran.txt

```diff
@@ -822,30 +822,47 @@ Extra - asset_id: collision_camera_011
   * Extra - left_shield_def: actordef:actors/props/door_shield_plasma/charclasses/door_shield_plasma.bmsad
   * Extra - right_shield_entity: {EMPTY}
   * Extra - right_shield_def: None
-  > Dock to Map Station Access (dooremmy_002)
+  > Dock to Map Station Access (Lower)
       Trivial
+  > Dock to EMMI Zone Exit West
+      All of the following:
+          # If you start in the room direclty below this one, you can carry a speed boost all the way to here
+          Morph Ball and Speed Booster and Speedbooster Conservation (Beginner) and Shoot Plasma Beam
```

Comment:
What's the reason for needing plasma beam here? This should be accounted for by the door

Reply:
So I think from this door, it should be fine to not have the plasma requirement, but not the other one.